x/delayedack/keeper/keeper.go

```diff
@@ -102,15 +102,14 @@ func (k Keeper) GetRollapp(ctx sdk.Context, chainID string) (rollapptypes.Rollap
 }
 
 func (k Keeper) GetRollappFinalizedHeight(ctx sdk.Context, chainID string) (uint64, error) {
-	res, err := k.rollappKeeper.StateInfo(ctx, &rollapptypes.QueryGetStateInfoRequest{
-		RollappId: chainID,
-		Finalized: true,
-	})
-	if err != nil {
-		return 0, err
+	// GetLatestFinalizedStateIndex
+	latestFinalizedStateIndex, found := k.rollappKeeper.GetLatestFinalizedStateIndex(ctx, chainID)
+	if !found {
+		return 0, rollapptypes.ErrNoFinalizedStateYetForRollapp
 	}
 
-	return res.StateInfo.StartHeight + res.StateInfo.NumBlocks - 1, nil
+	stateInfo := k.rollappKeeper.MustGetStateInfo(ctx, chainID, latestFinalizedStateIndex.Index)
```

Comment:
this will panic until first state info is written to the hub and I'm not sure it will show a nice error in the relayer in that case. maybe worth just getting the state info.

Reply:
how that? if `latestFinalizedStateIndex` found, there must be the state info
and not sure how relayer related, it's internal call of delayedack